### ISSUE TITLE
Non-text contrast ratio between keyboard focus and background of 'Edit' button is less than required 3:1 ratio.

### ISSUE THREAD

**Issue by msftedad** [ag-grid/ag-grid#7144]
**Repro steps:**

1. Open the URL [Stratus](https://sandbox.stratus.ms/stratuscontrols/readwritelayout)
2. Expand sample selection and navigate to controls
3. Verify luminosity ratio of keyboard focus indicator.

**Current behavior**
Non-text contrast ratio between focus and background of 'Edit' button is 1.3:1 which is less than 3:1.

**Expected behavior**
Non-text contrast ratio between focus and background of 'Edit' button should be greater than and equal to 3:1.

**Please tell us about your environment:**
Device: Windows 11 Enterprise22H2
Version: 22621.2428

* **AG Grid version:** X.X.X
26.1

* **Browser:**
Browser: Edge version 117.0.2045.60 (Official build) (64-bit)

* **Language:**
Typescript

**Attachment:**
![Non-text contrast ratio between keyboard focus and background of 'Edit' button is less than required 31 ratio](https://github.com/ag-grid/ag-grid/assets/93735775/4374f2ce-5317-4eba-bc39-83d6455c8dd1)

**Comment by AG-Zoheil** [comment]
Hi @msftedad,

Thank you for raising this on Github.

The link you've provided is hidden behind a login so I am unable to access it.

Can you clarify which component of ours it is you are using - if you could link to an example from our website that would be much appreciated.

Kind regards,
Zoheil

**Comment by AG-Zoheil** [comment]
Thank you for submitting this on Github.

As this has been open without any further interaction for a while, I will be closing this issue.

If the issue has already been resolved by other means, you can ignore this. If however, you are still in need of support you can choose to raise the issue again for the AG Grid community to support.

Kind regards,
Zoheil

**Comment by msftedad** [comment]
Hi @AG-Zoheil, Currently we have closed this issue. If we have any query we will log new issue or reopen this. Thanks
